src/main/java/com/google/sps/servlets/DisplayImage.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/get-image")
+public class DisplayImage extends HttpServlet {
```

Comment:
So far I've seen that "Servlet" is kept in the file and class name, so maybe it would be better to change the file & class name to "DisplayImageServlet"

Reply:
Hmm
Yes it seems so.
Okay I'll update that!